packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -118,21 +120,21 @@ const DashboardHeader = ({
                     <PageTitle>{dashboardName}</PageTitle>
 
                     {dashboardDescription && (
-                        <Tooltip2
-                            content={dashboardDescription}
-                            position="bottom"
-                        >
-                            <Button icon="info-sign" minimal />
-                        </Tooltip2>
+                        <Tooltip label={dashboardDescription} position="bottom">
+                            <ActionIcon color="dark">
```

Comment:
why is this dark?

Reply:
because looks better in `dark` than gray IMO

<img width="757" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/64b2c153-a959-462f-abcb-d7684cfc0d4c">

Also, @PriPatel will play around and might revert! thanks for pointing this out 👍

Reply:
changed from `ActionIcon` to `Icon` 🙌  thanks @PriPatel for the feedback